controllers/runner_graceful_stop.go

```diff
@@ -187,6 +210,11 @@ func ensureRunnerUnregistration(ctx context.Context, retryDelay time.Duration, l
 		}
 
 		if runnerBusy {
+			_, err := labelPod(ctx, c, log, pod, AnnotationKeyUnregistrationFailureMessage, runnerUnregistrationFailureMessage)
```

Comment:
hi, @mumoshu 
this line introduces an error like:
```
Failed to patch pod to have actions-runner/unregistration-failure-message annotation","runnerpod":"actions-runner/databend-nqjfv-gvv7v","error":"Pod \"databend-nqjfv-gvv7v\" is invalid: [metadata.labels: Invalid value: \"Bad request - Runner \\\"databend-nqjfv-gvv7v\\\" is still running a job\\\"\": must be no more than 63 characters, metadata.labels: Invalid value: \"Bad request - Runner \\\"databend-nqjfv-gvv7v\\\" is still running a job\\\"\": a valid label must be an empty string or consist of alphanumeric characters, '-', '_' or '.', and must start and end with an alphanumeric character (e.g. 'MyValue',  or 'my_value',  or '12345', regex used for validation is '(([A-Za-z0-9][-A-Za-z0-9_.]*)?[A-Za-z0-9])?')]",
```

Reply:
@everpcpc Ahhh good catch! I also realized this while testing it before 0.25.0 release but I lost my memory since then.
At least this doesn't necessarily be a full error message, so truncating it to 63 characters should be better than the current state.
I'm also wondering if we need to strip some characters that aren't valid within an annotation value.
I'm going to be busy for a few days- Would you mind submitting a pull request to fix it, so that I can review it and cut a new release asap? Thanks!

Reply:
ref: https://github.com/actions-runner-controller/actions-runner-controller/pull/1615